.github/workflows/test_all.yml

```diff
@@ -13,14 +14,11 @@ jobs:
       - name: python environment step
         uses: actions/setup-python@v5
         with:
```

Comment:
Even this can possibly be removed then. Shell file won't need python I think.

Non-blocking, as hopefully we are planning to add these back when we find a way.

Reply:
yes, that was my idea - leaving the "skeleton" there for the hopefully close time where we know how to fix it.